data/commodities.txt

```diff
@@ -709,30 +847,44 @@ trade
 		"volatile gases"
 		"waste"
 	commodity "Construction"
+		"bolts"
 		"box girders"
 		"carbon steel"
+		"cement"
 		"composite beams"
 		"composite fibers"
 		"concrete"
+		"drills"
 		"duct tape"
 		"fiberglass"
+		"French windows"
 		"girders"
+		"gravel"
 		"I-beams"
+		"jackhammers"
 		"joint plates"
 		"lumber"
 		"metal joists"
 		"metal trusses"
+		"nails"
+		"nuts"
 		"pylons"
+		"reinforced steel"
```

Comment:
Reinforced steel isn't a thing in itself, that's rebar, which is already on the list.

Reply:
It can be arguably moved to construction as a type, but reinforced steel doesn't make sense, as it's the steel that does the reinforcing. So it's "reinforcing steel", which is called rebar in practice: https://www.google.com/search?client=firefox-b-lm&q=reinforced+steel